ExceptionHandling/Task1/Program.cs

```diff
@@ -1,10 +1,29 @@
-﻿namespace Task1
+﻿using System;
+
+namespace Task1
 {
     internal class Program
     {
         private static void Main(string[] args)
         {
-            // TODO: Implement the task here.
+            while (true)
+            {
+                Console.WriteLine("\nEnter the line: ");
+                var line = Console.ReadLine();
+
+                try
+                {
+                    Console.WriteLine(line[0]);
+                }
+                catch (IndexOutOfRangeException ex)
+                {
+                    Console.WriteLine("Input string was null.");
```

Comment:
could you please provide the value of input when it is `null`

Reply:
Fixed